.travis.yml

```diff
@@ -10,8 +10,9 @@ before_install:
 
 env:
   matrix:
-    - INSTANCE: deb-debian-9
-    - INSTANCE: deb-ubuntu-1804
+    - INSTANCE: debian-debian-9
+    - INSTANCE: ubuntu-ubuntu-1604
+    - INSTANCE: ubuntu-ubuntu-1804
     - INSTANCE: centos-centos-7
```

Comment:
How do you feel about adding all of the rest of the platforms that we've got working on `template-formula`?

Reply:
Added them.